css/main.css

```diff
@@ -198,25 +235,93 @@ a.logo{
 .breadcrumb > .active {
   color: #f03c3c;
 }
-
+.container-blog-gridzilla{
+    padding-left: 18px;
+    padding-right: 20px;
+}
+.col-blog-gridzilla{
+    padding-left: 10px;
+    padding-right: 10px;    
+}
 /* Article_post*/
 .post{
-    padding-bottom: 0 10px 10px 10px;
+    padding-bottom: 21px;
 }
 .post-imgWrapper img{
     width: 100%;
 }
-.post-hover:hover{
-    background-color: rgba(0, 0, 0, 0.75);
+.post-imgWrapper{
+    position: relative;
+}
+.post-imgWrapper:hover .post-hover{
+    display: list-item;
+}
+.post-hover{
+    position: absolute;
+    top: 0;
+    left: 0;
     height: 100%;
+    width: 100%;
```

Comment:
I think you can replace `height: 100%; width: 100%;` with `bottom: 0; right: 0;`